client/src/components/JurisdictionAdmin/BatchRoundSteps/TallyEntryAccountsStep.tsx

```diff
@@ -142,6 +145,14 @@ const TurnOnTallyEntryAccountsPrompt: React.FC<ITurnOnTallyEntryAccountsPromptPr
   )
 }
 
+const CodeInput = styled(CodeInputAtom)`
+  input {
+    width: 70px;
+    font-size: 50px;
+    height: 80px;
+  }
```

Comment:
Instead of specifying this style here, we could bake these styles into `CodeInput`, toggle-able via an optional size prop, e.g. `size="large"` or just `large`

Reply:
I thought about that. I don't have a strong opinion, but thought it might be nice to see what size was desired in other use cases before building any size styling into the component.

Reply:
(I'm specifically thinking of the jurisdiction admin login screen as the next potential use case)

Reply:
Seems reasonable to me!

Reply:
I think I tend to err on the side of baking logic into components to encourage consistency, even if it's not yet clear what the consistent standard is that we want to settle on

Reply:
That makes sense as an approach to maintain visual consistency especially. Though on the other hand, it can make it harder to modify later without breaking consumers. I think I'll leave this one for now, but will noodle on that more.